### DISCUSSION TITLE
Where best to put local repo in mirrored production

### DISCUSSION THREAD

**Discussion by SRFTInterface** [intersystems/git-source-control#343] [Q&A]
Hi:
We are ready to take our GIT to the next stage and trial in our mirrored preprod production (1 server DC1 1 server DC2 1 server 1 Abiter DC3). We were thinking it is best to go on a file on the arbiter drive? 

What happens if Arbiter is down/ Local repo inaccessible?

Asked as a Q&A as think it's a common question of the deployment on shared environment/ live mirrored setup

**Comment by isc-tleavitt** [comment]
Sorry for the slow repsonse here. I started to look around for best practices and forgot to circle back.

It's a fantastic question, and I think your gut feeling is correct - the local-to-the-server repo should be in a place accessible from all mirror members, provided you can do this in a way that doesn't introduce a single point of failure operationally.

If that location is unavailable, you won't be able to do development, but operations on the running instance shouldn't be impacted otherwise (and that location being unavailable would be something that needs to be fixed immediately anyway).
